libs/portals/core/src/screens/ModuleRoute.tsx

```diff
@@ -1,54 +1,45 @@
 import React, { Suspense, useEffect } from 'react'
 import { useLocation } from 'react-router-dom'
-import { User } from '@island.is/shared/types'
-import { ModuleErrorScreen, ModuleErrorBoundary } from './ModuleErrorScreen'
+import { ModuleErrorScreen } from './ModuleErrorScreen'
 import { PortalRoute } from '../types/portalCore'
 import { usePortalMeta } from '../components/PortalProvider'
 import { plausiblePageviewDetail } from '../utils/plausible'
 import { Box } from '@island.is/island-ui/core'
 
 type ModuleRouteProps = {
   route: PortalRoute
-  userInfo: User
 }
 
-export const ModuleRoute = React.memo(
-  ({ route, userInfo }: ModuleRouteProps) => {
-    const location = useLocation()
-    const { basePath } = usePortalMeta()
+export const ModuleRoute = React.memo(({ route }: ModuleRouteProps) => {
+  const location = useLocation()
+  const { basePath } = usePortalMeta()
 
-    useEffect(() => {
-      if (route.render !== undefined) {
-        plausiblePageviewDetail({
-          basePath,
-          path: route.path,
-        })
-      }
-    }, [basePath, location, route.path, route.render])
-
-    if (route.render === undefined) {
-      return null
+  useEffect(() => {
+    if (route.element !== undefined) {
+      plausiblePageviewDetail({
+        basePath,
+        path: route.path,
+      })
     }
+  }, [basePath, location, route.path, route.element])
 
-    const Module = route.render({
-      userInfo,
-    })
+  if (!route.element) {
+    return null
+  }
 
-    if (Module)
-      return (
-        <Suspense fallback={null}>
-          <Box paddingY={1}>
-            <ModuleErrorBoundary name={route.name}>
-              <Module userInfo={userInfo} />
-            </ModuleErrorBoundary>
-          </Box>
-        </Suspense>
-      )
+  const module = route.element
 
+  if (module) {
     return (
-      <Box paddingY={1}>
-        <ModuleErrorScreen name={route.name} />
-      </Box>
+      <Suspense fallback={null}>
+        <Box paddingY={1}>{module}</Box>
+      </Suspense>
     )
-  },
-)
+  }
```

Comment:
Checking `route.element` twice? Do we even need this check? Can probably get rid of the ModuleErrorScreen in this case.